_posts/0000-01-02-Toniyen.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
**Mr. Robitson, calm yourself man! The Lion must go back to its den, prime minister**